mala/datahandling/data_converter.py

```diff
@@ -346,14 +360,18 @@ def __convert_single_snapshot(self, snapshot_number,
                             input_mesh_component.reset_dataset(dataset)
 
                             # TODO: Remove this copy?
-                            input_mesh_component.\
-                                store_chunk(tmp_input[:, :, :, i].copy())
+                            input_mesh_component[:] \
+                                = tmp_input[:, :, :, i].copy()
```

Comment:
Doing a copy here is probably the most efficient thing possible, since this code indexes the smallest dimension and the uncopied buffer consists just of single values scattered across memory. Better to collect them into a proper buffer than to write n little values to the file system separately.

Reply:
discussed offline: need to check the behavior of numpy reorganize operations, might reorganize the dataset before storing off to memory